.github/workflows/e2e-run.yml

```diff
@@ -134,7 +134,7 @@ jobs:
   reports:
     runs-on: ubuntu-latest
     needs: tests
-    if: ${{ !cancelled() && github.repository == 'kafbat/kafka-ui' }}
+    if: ${{ !cancelled() && github.event.pull_request.head.repo.full_name == github.repository }}
```

Comment:
One more thing, out of three workflows we have which call this one, only one of them has `pull_request` trigger. How will this work for two others (`workflow_dispatch` and `schedule`)?

Reply:
it will not run the reports task, do we want to make the task still run for two other workflow?

Reply:
I already change the condition, so now the reports task will run if not cancelled and get trigger not from pull request org trigger from pull request that not from forks